simulation.go

```diff
@@ -250,12 +244,13 @@ type SimulationBFTree struct {
 // 'addresses'. The network.Address(es) created are of type TLS or PlainTCP,
 // depending on the value 'TLS' in 'sc'.
 func (s *SimulationBFTree) CreateRoster(sc *SimulationConfig, addresses []string, port int) {
+	if sc.Builder == nil {
+		// TODO: must have a better way to inforce to get the builder.
```

Comment:
`SimulationConfig` and `SimulationBFTree` are quite close, maybe creating the latter with help of the first (thus ensuring to have a builder)?

Reply:
They even both have a `TLS` key, with the same meaning.